massgov/mayflower
# Feature/markie/dp 10242  checkbox

## Description
Copies logic of checkbox input from patternlab to react.

## Steps to Test
[ ] - Checkout branch
[ ] - npm run start
[ ] - Navigate to http://localhost:6006/?selectedKind=atoms%2Fforms&selectedStory=InputCheckBox&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel
[ ] - Observe checkbox item

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
![image](https://user-images.githubusercontent.com/7987464/45122157-4e911180-b120-11e8-8254-43a931c77309.png)

#### Today I learned...
Checkboxes need to set state on change otherwise they are not interactive in react.
